src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -47,6 +50,11 @@ public class Drive extends LoggedSubsystem<SwerveDriveData, SwerveDriveMap> {
     double maxRotationRadiansPerSecond;
     double speedCoef = 1;
     double rotationCoef = 1;
+    // Both of these are so wrong
```

Comment:
Concerning...

Reply:
"For vision stuff it doesn't matter wont affect the robot" -Tim